package.json

```diff
@@ -966,8 +966,8 @@
             }
           },
```

Comment:
i think we need to update the description for `"go.useLangaugeServer"`. currently it only refers to the Sourcegraph go language server

Reply:
Done